src/tools/autocompletion/find_autocompletion_options.py

```diff
@@ -137,12 +139,56 @@ def complete_commands(start_of_input):
 # searches for options starting with start_of_input belonging to last_word
 # return: list of all possible arguments
 def complete_options():
-	global start_of_current_input, last_command
+	global last_command
+	completion_options = []
+	if last_command is not None:
+		completion_options.extend(complete_options_opt_array(last_command))
+	else:
+		completion_options.extend(complete_options_single())
+	return completion_options
+
+def complete_options_single():
```

Comment:
Please document what this function should do: should it handle both single options in array and non-arrays?